MultisigIntegrationTests/UI Snapshot Tests/BalanceTableViewCellTests.swift

```diff
@@ -0,0 +1,48 @@
+//
+//  BalanceTableViewCellTests.swift
+//  MultisigIntegrationTests
+//
+//  Created by Dmitry Bespalov on 09.06.21.
+//  Copyright © 2021 Gnosis Ltd. All rights reserved.
+//
+
+import XCTest
+import SnapshotTesting
+@testable import Multisig
+
+class BalanceTableViewCellTests: XCTestCase {
+    func testEthValue() {
+        let vc = CellTestViewController<BalanceTableViewCell>(estimatedHeight: 60) { cell in
+            cell.setMainText("ETH")
+            cell.setDetailText("0,005")
+            cell.setSubDetailText("155,84 USD")
+            cell.setImage(UIImage(named: "ico-ether")!)
+        }
+        assertSnapshot(matching: vc, as: .image(on: .iPhoneX))
+    }
+
+    func testNotConfigured() {
```

Comment:
do we need to test notConfigured/Empty at all?

Reply:
These are testing edge cases. Not configured is used because the XIB had some text, so I fixed it. Empty was used to double-check that it works with empty values.